### PR TITLE
Update to use intl tool, fix import error

### PR DESCRIPTION
**Pull request by huynguyennovem**
- Update to use intl tool
- Fix import error

Signed-off-by: Huy Nguyen <qhuynguyen@linagora.com>

### CODE REVIEW

**Review by imGok — APPROVED** [review]

**Comment by hoangdat** [comment]
Lgtm

**Review by hoangdat — APPROVED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -155,3 +155,5 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+flutter_intl:
+  enabled: true
```
Comment (imGok) [inline pubspec.yaml:159]: Don't you need to install intl_utils in order to use this ?

Comment (huynguyennovem) [reply pubspec.yaml:159]: @imGok : this is [IDE's plugin](https://plugins.jetbrains.com/plugin/13666-flutter-intl) which supports to generate boilerplate code for Intl. Have a talk a look at this [Notice](https://pub.dev/packages/intl_utils#usage) also :)

Comment (imGok) [reply pubspec.yaml:159]: Oh sure ! Thanks for docs

**Comment by hoangdat** [comment]
not compatible with CI -> closed

**Pull request closed without merging**